### DISCUSSION TITLE
Why are some tabs indented?

### DISCUSSION THREAD

**Discussion by vuslatx** [mbnuqw/sidebery#692] [Q&A]
![6012](https://user-images.githubusercontent.com/65629417/178569990-00ae9c9c-056c-45bb-ace7-8fe0e19beefe.png)

Tab with arrow is intended(with space). Why? What does it mean?

**Comment by niun** [comment]
This is a big feature of Sidebery: Organizing the tabs in a tree.

You propably opened the indented tab from the tab above, so it is associated with this tab where you opened it from.

When clicking on the Icon of the tab above, it should fold the indented tab below. This way you can organize Tabs like in Folders. Try to right click a tab and select "Group". You can drag n drop tabs onto other tabs or groups, to put them in its tree branch and then fold it away for a better overview.

**Reply by vuslatx** [reply]
thanks but i don't want this feature. Is there any way to disable this? I want one tree of all tabs.

**Reply by albino1** [reply]
Just guessing, but I imagine in the settings under `Tabs Tree` you can just turn `Tabs Tree Structure` off.
